crates/driver/src/infra/blockchain/mod.rs

```diff
@@ -131,10 +131,18 @@ impl Ethereum {
         &self.inner.current_block
     }
 
+    fn max_block_size(&self) -> eth::U256 {
+        const ARBITRUM_ONE_CHAIN_ID: u64 = 42161;
+
+        match self.inner.chain.0 {
+            ARBITRUM_ONE_CHAIN_ID => 500_000_000,
```

Comment:
Is this correct? According to https://docs.arbitrum.io/build-decentralized-apps/reference/chain-params the arbitrum max block size is 32M

If all we need is a custom gas price for simulation/access list, maybe this should be something we configure the binary with another parameter (with default being 30M). 

I remember for mainnet for instance some clients rejected simulation requests where the gas limit was set to something larger than the block gas limit (whereas some other clients where fine with setting it to `null` meaning "whatever gas you need")

Reply:
Additionally, has this been checked on the arbitrum RPC nodes that we are running? Looking at the default nitro node configuration shows that there is a gas limit of 50M on `eth_call` RPC calls. To see this, assuming you have docker available:

```bash
docker run offchainlabs/nitro-node:v2.3.2-064fa11 --help
```

Reply:
I wouldn't be surprised if this had to be `null` to work (meaning use whatever max is possible without looking at L1 calldata cost)...